packages/slider/src/spectrum-config.js

```diff
@@ -51,6 +69,24 @@ const config = {
                     ],
                     'variant'
                 ),
+                {
```

Comment:
Can this change come from spectrum-css? My only goal here is to keep spectrum-config as light as possible